python/storage/pyproject.toml

```diff
@@ -0,0 +1,47 @@
+[tool.poetry]
+name = "storage"
+version = "0.14.0"
+description = "A storage initializer for KServe"
```

Comment:
I am confused of this description because it is the same as this(https://github.com/kserve/kserve/pull/3902/files#diff-932ce623b0d496fa19859e372bc304d7f01400e0fc1e575c36c2959d700e03acR4)
it would be better to give more detail description to distinguish with the others